### DISCUSSION TITLE
Stuck on "Advanced Usage of Pake"

### DISCUSSION THREAD

**Discussion by pseudofractal** [tw93/Pake#485] [Q&A]
I was trying to rewrite the styling of site, when I got stuck on trying to figure out what the following line meant:

> and find the location of the style in inject/pake.css with style.innerHTML

The closest thing I can find using a search is the file [`src-tauri/src/inject/style.js`](https://github.com/tw93/Pake/blob/4fc84a2dfd06f8808f49c0a5051af726a7350c54/src-tauri/src/inject/style.js).
Are the docs a bit outdated at this point in time, or am I confusing it for something else?

On top of this, I also can not find any of the examples mentioned in:

> there are some examples you can copy

A link to these would be great.

The other discussion [#247] that I could find asking similar questions, also seems to have fizzled out.

Thank you for reading :D

[Link to the advanced usage page for quick jumping](https://github.com/tw93/Pake/wiki/Advanced-Usage-of-Pake)

**Comment by jeasonnow** [comment]
@PsuedoFractal  Thank you for your suggestion. The documentation has now been updated according to the latest code.
